packages/main/src/webComponents/IllustratedMessage/index.tsx

```diff
@@ -178,7 +187,7 @@ const IllustratedMessage = withWebComponent<IllustratedMessagePropTypes, Illustr
   'ui5-illustrated-message',
   ['accessibleNameRef', 'name', 'size', 'subtitleText', 'titleText'],
   [],
-  ['subtitle'],
+  ['subtitle', 'title'],
```

Comment:
This must be removed, we don't support the `title` slot for the IllustratedMessage.